content/en/agent/kubernetes/distributions.md

```diff
@@ -143,7 +143,9 @@ spec:
 {{% /tab %}}
 {{< /tabs >}}
 
-**Notes**: 
+**Notes**:
+
+- Since Agent 7.35, tlsVerify: false is currently required as Kubelet certificates in AKS do not have any SAN (Subject Alternative Name) set.
```

Comment:
```suggestion
- As of Agent 7.35, `tlsVerify: false` is required because Kubelet certificates in AKS do not have a Subject Alternative Name (SAN) set.
```